generators/go/sdk/versions.yml

```diff
@@ -1,3 +1,10 @@
+- version: 1.0.0
+  changelogEntry:
+    - summary: |
+        Major version release to update the generator's default set of configuration options.
```

Comment:
I would add the old and new options in the changelog so people can find it easily

Reply:
Good call, added!